### ISSUE TITLE
Fix PyTorch warnings

### ISSUE THREAD

**Issue by simondlevy** [simondlevy/ES-Gym#1]
/usr/local/lib/python3.6/dist-packages/gym/envs/registration.py:14: PkgResourcesDeprecationWarning: Parameters to load are deprecated.  Call .resolve and .require separately.
  result = entry_point.load(False)
/usr/local/lib/python3.6/dist-packages/torch/nn/modules/container.py:100: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.

**Comment by simondlevy** [comment]
Remaining warning was fixed by sudo pip3 install --upgrade gym
